src/olympia/addons/utils.py

```diff
@@ -84,31 +86,36 @@ def get_featured_ids(app, lang=None, type=None):
 
 
 @memoize('addons:creatured', time=60 * 10)
-def get_creatured_ids(category, lang):
+def get_creatured_ids(category, lang=None):
```

Comment:
It's a wordplay on `featured` + `category`. Threw me off initially but all the code uses it - from the views to the comments, the template name, the model methods and even the url names...